### DISCUSSION TITLE
upload上传成功后怎么禁止拖拽上传？

### DISCUSSION THREAD

**Discussion by 1679812625** [layui/layui#2038] [问答]
目前我通过设置 disabled属性 实现了禁止点击上传
但是还是可以拖拽上传 有没有什么办法禁止？

**Comment by hjingsuper** [comment]
[重载](https://layui.dev/docs/2/upload/#reload) `drag`属性应该可以吧

```
  // 重载
  inst.reload({
    drag：false
  })

```

**Comment by 1679812625** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法及时回复你的邮件。我将尽快给你回复。
